freva-rest/src/freva_rest/databrowser_api/endpoints.py

```diff
@@ -294,7 +293,8 @@ async def load_data(
     [!NOTE]
     The urls are only temporary and will be invalidated.
     """
-    if "zarr-stream" not in os.getenv("API_SERVICES", ""):
+    if "zarr-stream" not in server_config.services:
+        print("foo", server_config.services)
```

Comment:
Perhaps replace the `print` statement with a `logger.exception` statement?

Reply:
Good, deleted.